hdat/archive.py

```diff
@@ -13,16 +11,24 @@ def __init__(self, directory):
         os.makedirs(self.root, exist_ok=True)
 
     def select(self, suite_id, case_id, result_id):
+        """Return a single result specified"""
         result_filename = self._result_filename(suite_id, case_id, result_id)
 
         if not os.path.isfile(result_filename):
             return None
         else:
             return self.read_result(result_filename)
 
-    def select_recent(self, i, *args):
+    def select_recent(self, suites, i, *args):
+        """Return the most recent result of a specified case.
```

Comment:
We should follow PEP8 and use either:

def foo():
    """A multi-line
    docstring.
    """

OR
def bar():
    """
    A multi-line
    docstring.
    """

Reply:
I personally like the second form better

Reply:
For `select`, which has a single line docstring, would it be reasonable to use the same format as the other multi-line docstrings for consistency?

Reply:
![image](https://user-images.githubusercontent.com/9055029/43593672-0a410e42-963e-11e8-9517-d24e680e3a52.png)

Reply:
Agreed; this is what I like to do personally--other people like using just one line.  I think either way is fine.